### PR TITLE
add funcionality to be able to log out

### PR DESCRIPTION
**Pull request by TolozaLeo**
i made the funcionality mencioned and add a 3 seconds pause to the transsition to see the loading animation :D.

### CODE REVIEW

**Review by NahuelToloza — COMMENTED** [review]
Bien! Revisar los comentarios

File: app/src/main/java/com/example/pokemonclasses/presentation/ui/fragments/home/menu/LogoutFragment.kt [hunk 1]
```diff
@@ -19,6 +30,24 @@ class LogoutFragment: Fragment() {
...
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        updateData()
+        navigateToLogginFragment()
+    }
+
+    private fun navigateToLogginFragment() {
+        val action = LogoutFragmentDirections.actionLogoutFragmentToLoginFragment()
+        Handler(Looper.getMainLooper()).postDelayed({
```
Comment (NahuelToloza) [inline app/src/main/java/com/example/pokemonclasses/presentation/ui/fragments/home/menu/LogoutFragment.kt:41]: Agregar //TODO

**Comment by TolozaLeo** [comment]
its not finish yet.

**Comment by TolozaLeo** [comment]
Now its finished, the suggestions by Nahuel was implemented and the conflicts with branch master solved.

**Review by NahuelToloza — APPROVED** [review]
Buena!!

**Pull request merged**